packages/components/src/tools-panel/test/index.js

```diff
@@ -977,37 +985,7 @@ describe( 'ToolsPanel', () => {
 			);
 
 			const optionsDisplayedIcon = screen.getByRole( 'button', {
-				name: 'View options',
-			} );
-
-			expect( optionsDisplayedIcon ).toBeInTheDocument();
-		} );
-
-		it( 'should render appropriate icons for the dropdown menu where there are no default controls', async () => {
```

Comment:
I've removed this test because the button label is no longer 'View add options' when the tools panel has all items disabled. It's now always the same so that users know it's the same menu.

I did try changing the test to reason about the aria-description that I added when all options are hidden, but react testing library doesn't seem to be very good at query elements by their description, despite it being documented. 🤷

Reply:
Managed to solve this using `expect( element ).toHaveAccessibleDescription( descriptionText )`.